pyproject.toml

```diff
@@ -0,0 +1,94 @@
+[build-system]
+requires = ["flit-gettext", "flit_scm", "wheel"]
+build-backend = "flit_gettext.scm"
+
+[project]
+name = "django-hijack"
+authors = [
+  { name = "arteria GmbH", email = "admin@arteria.ch" },
+  { name = "Johannes Maron", email = "johannes@maron.family" },
+]
+readme = "README.md"
+license = { file = "LICENSE" }
+keywords = ["Markdown", "Django", "email", "templates", "HTML"]
+dynamic = ["version", "description"]
+classifiers = [
+  "Development Status :: 5 - Production/Stable",
+  "Environment :: Web Environment",
+  "License :: OSI Approved :: MIT License",
+  "Intended Audience :: Developers",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3 :: Only",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: 3.12",
+  "Framework :: Django",
+  "Framework :: Django :: 4.2",
+  "Framework :: Django :: 5.0",
+]
+requires-python = ">=3.8"
+dependencies = ["django>=4.2"]
+
+[project.optional-dependencies]
+test = [
+  "pytest",
+  "pytest-cov",
+  "pytest-django",
+]
+lint = [
+  "bandit==1.7.8",
+  "black==24.4.2",
+  "flake8==7.0.0",
```

Comment:
Coul you please consider ruff as a replacement for flake8 bandit isort, and probably black?
https://docs.astral.sh/ruff/

Reply:
Yes, I was thinking about it too. I'll make the change. It'll also make the setup.cfg file fully obsolete.